requirements/customer/introduction.tex

```diff
@@ -0,0 +1,3 @@
+\section{Introduction}
+
+The JobAdder project enables users to remotely execute resource-intensive tasks on a dedicated cluster of work machines. Tasks are automatically distributed optimally, in order to achieve lower system load and shorter execution times. In addition, JobAdder provides tools to inspect the state of the whole system and display statistics about user activity.
```

Comment:
>Tasks are automatically distributed optimally, in order to achieve lower system load and shorter execution times.

I think saying that our distribution of tasks is optimal is strictly speaking incorrect because we do not know in advance how long the tasks will run.
Also I think that "system load" is not sufficiently precise.
I suggest the following wording:

Tasks are automatically being distributed to work machines.
The distribution reduces task execution times and peak system loads by better utilizing the available hardware.

Reply:
I'd say "the distribution tries to reduce" but overall your wording is better.